documentation/en/cli-lotus.md

```diff
@@ -10,11 +11,12 @@ VERSION:
    1.31.1-dev
 
 COMMANDS:
-   daemon   Start a lotus daemon process
-   backup   Create node metadata backup
-   config   Manage node config
-   version  Print version
-   help, h  Shows a list of commands or help for one command
+   daemon        Start a lotus daemon process
```

Comment:
do we have a minimum width setting here that needs adjusting? looks like we have 4 extra spaces than we need